hooks/useConnectedAppConfig.ts

```diff
@@ -0,0 +1,35 @@
+import {useEffect, useState} from "react"
+
+export type ConnectedAppConfig = {
+  app: {
+    icon: string
+    title: string
+  }
+  services: {"OpenID.scopes": string}
+  type: string
+}
+
+export default function useAppConfig() {
+  const [connectedAppConfig, setConnectedAppConfig] =
+    useState<ConnectedAppConfig | null>(null)
+
+  useEffect(() => {
+    function callback(e: {data: ConnectedAppConfig}) {
+      if (typeof e.data !== "object") return
+      if (e.data.type !== "FCL:AUTHN:CONFIG") return
```

Comment:
This is moved over from https://github.com/onflow/fcl-dev-wallet/pull/30/files#diff-295f53ab25aad310443cdc8b33f50b7c65cbdfe400452c9bfada4de4276c61ddL273 . The config starts off as null, and only gets set when the app receives a FCL:AUTHN:CONFIG event.

The Null Config notice was previously commented out in https://github.com/onflow/fcl-dev-wallet/pull/30/files#diff-295f53ab25aad310443cdc8b33f50b7c65cbdfe400452c9bfada4de4276c61ddL289 , but I added it back in the AppContext.

I can comment it out again so that the config and accounts list is visible when accessing the dev wallet directly.

Reply:
Sorry, my comment is missing some detail. Authentication `onReady` sends `type: "FCL:AUTHN:CONFIG"` so `setConnectedAppConfig(e.data)` sets in that case. However, when authorizing a mutation/transaction the iFrame onReady sends `type: "FCL:FRAME:READY:RESPONSE"` so no AppConfig data is not set in context

Reply:
Ahh that makes sense, thanks! Do you mind if I handle that as part of the authorization UI updates? I'll be getting started after the edit/create account pages.

Reply:
Following up on this - it was addressed in #37 by separating the authz and authn contexts